data/prod/news/articles/c85pqyj5m2ko.json

```diff
@@ -1,756 +1,801 @@
-          "curationType": ["vivo-stream"],
-          "thingId": "803eaeb9-c0c3-4f1b-9a66-90efac3df2dc",
-          "thingLabel": "Duchess of Sussex",
-          "thingType": ["Thing", "Person"]
-        },
-        {
-          "thingUri": "http://www.bbc.co.uk/things/282061c0-3743-4eb0-9dde-6d710389630f#id",
-          "topicId": "c8zwn8k9nlet",
-          "topicName": "Windsor",
-          "curationType": ["location-stream"],
-          "thingId": "282061c0-3743-4eb0-9dde-6d710389630f",
-          "thingLabel": "Windsor",
-          "thingType": ["Thing", "Place"]
-        },
-        {
-          "thingUri": "http://www.bbc.co.uk/things/f2815ef8-ea42-414b-865c-d5e92b5207e7#id",
-          "topicId": "cy4ng11dgygt",
-          "topicName": "Hackney",
-          "curationType": ["location-stream"],
-          "thingId": "f2815ef8-ea42-414b-865c-d5e92b5207e7",
-          "thingLabel": "Hackney",
-          "thingType": ["Thing", "Place"]
-        }
-      ],
-      "mentions": []
-    }
-  },
   "content": {
     "model": {
       "blocks": [
         {
-          "type": "headline",
           "model": {
             "blocks": [
               {
-                "type": "text",
                 "model": {
                   "blocks": [
                     {
-                      "type": "paragraph",
                       "model": {
-                        "text": "Royal wedding flowers given to Hackney hospice",
                         "blocks": [
                           {
-                            "type": "fragment",
                             "model": {
-                              "text": "Royal wedding flowers given to Hackney hospice",
-                              "attributes": []
-                            }
+                              "attributes": [],
+                              "text": "Royal wedding flowers given to Hackney hospice"
+                            },
+                            "type": "fragment"
                           }
-                        ]
-                      }
+                        ],
+                        "text": "Royal wedding flowers given to Hackney hospice"
+                      },
+                      "type": "paragraph"
                     }
                   ]
-                }
+                },
+                "type": "text"
               }
             ]
-          }
+          },
+          "type": "headline"
         },
         {
-          "type": "image",
           "model": {
             "blocks": [
               {
-                "type": "rawImage",
                 "model": {
-                  "width": 1024,
-                  "height": 576,
-                  "locator": "E7DB/production/_101655395_paulineclayton.jpg",
-                  "originCode": "cpsprodpb",
-                  "copyrightHolder": "St Joseph's Hospice"
-                }
+                  "copyrightHolder": "St Joseph's Hospice",
+                  "height": 360,
```

Comment:
Why has the image height and width been changed?

Reply:
/shrug :P this is just wants on test